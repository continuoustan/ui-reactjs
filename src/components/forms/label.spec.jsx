import React from 'react';
import renderer from 'react-test-renderer';

import Label from './label';

test('Label Display', () => {
    const tree = renderer.create(
        <Label htmFor='element' isRequired={true} >Hallo Welt</Label>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
