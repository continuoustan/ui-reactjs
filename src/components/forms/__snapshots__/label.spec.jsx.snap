// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Display 1`] = `
<label
  className="label--required"
  htmlFor={undefined}
>
  Hallo Welt
</label>
`;
